src/components/Message/Message.js

```diff
@@ -1,366 +1,300 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { TouchableOpacity } from 'react-native';
-import deepequal from 'deep-equal';
```

Comment:
I think with this no longer being used you can remove `deep-equal` from the root `package.json` deps

Reply:
i'll trim em out

Reply:
looks like we have file-loader and path in the styleguide.config, but removed the deeps